sky/global_user_state.py

```diff
@@ -826,8 +826,13 @@ def get_storage_names_start_with(starts_with: str) -> List[str]:
     return [row[0] for row in rows]
 
 
+def get_storage_names() -> List[str]:
+    rows = _DB.cursor.execute('SELECT name FROM storage')
+    return [row[0] for row in rows]
```

Comment:
can we use sth like `get_glob_storage_name('*')` to replace this function?

Reply:
Good catch!